src/helper/Constants.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
+
+import {IPriceFeed} from "../interfaces/IPriceFeed.sol";
+
+IPriceFeed constant ETH_PER_WEETH_CHAINLINK = IPriceFeed(0x5c9C449BbC9a6075A2c061dF312a35fd1E05fF22);
+IPriceFeed constant ETH_PER_EZETH_CHAINLINK = IPriceFeed(0x636A000262F6aA9e1F094ABF0aD8f645C44f641C); 
+IPriceFeed constant ETH_PER_RSETH_CHAINLINK = IPriceFeed(0x03c68933f7a3F76875C0bc670a58e69294cDFD01);
+IPriceFeed constant ETH_PER_RSWETH_CHAINLINK = IPriceFeed(0xb613CfebD0b6e95abDDe02677d6bC42394FdB857);
+IPriceFeed constant ETH_PER_PUFETH_CHAINLINK = IPriceFeed(0x76A495b0bFfb53ef3F0E94ef0763e03cE410835C);
```

Comment:
looks good, on chain how do we verify the underlying asset address with a redstone feed? there is no description I saw, and I am just kind of assuming that is correct feed for pufEth from their docs

Reply:
Yeah I was thinking about comparing the `description` variable (i.e. weETH/ETH for Chainlink), but Redstone unfortunately doesn't. 

So this is a source of human error.

Reply:
This check for the description is now implemented